Don't get rid of old symbols, that are exported. Other stuff depends on them
(eg PHP).
diff -u -r mhash-0.9.3.new/lib/mhash.c mhash-0.9.3/lib/mhash.c
--- lib/mhash.c	2005-12-07 03:55:30.000000000 +0000
+++ lib/mhash.c	2005-12-26 11:16:10.000000000 +0000
@@ -623,6 +623,12 @@
 	return(ret);
 }
 
+WIN32DLL_DEFINE void mhash_free(void *ptr)
+{
+        mutils_free(ptr);
+}
+
+
 /*
   Saves the state of a hashing algorithm such that it can be
   restored at some later point in time using
